lldb/include/lldb/Target/StackFrameList.h

```diff
@@ -91,7 +91,7 @@ class StackFrameList {
 
   size_t GetStatus(Stream &strm, uint32_t first_frame, uint32_t num_frames,
                    bool show_frame_info, uint32_t num_frames_with_source,
-                   bool show_unique = false,
+                   bool show_unique = false, bool should_filter = true,
```

Comment:
Can we rename `should_filter` to `show_hidden`. The former sounds like we're just about to perform the filtering whereas the latter sounds like some frames have a "hidden" flag and we should show them based on that. Also that would make it more consistent with the other flags.